### DISCUSSION TITLE
cmp finds completion , then doesn't , then doesn't again

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [hrsh7th/nvim-cmp#636] [Q&A]
https://user-images.githubusercontent.com/61395246/145691033-d5f1f03e-f7dc-4562-ac79-f39f0415b31b.mp4

Is this somehow my fault ? Or is this just normal behavior ?
Thanks!
PS. If this is not normal , I will follow up with config details and more.

**Comment by chriztheanvill** [comment]
Oh my friend, I thought it was just me. This drives me crazy.
Did you find out how to resolve this?

**Reply by chriztheanvill** [reply]
Sorry, just tried a few times, but does not solve it.
I think I found the solution: [cmp-fuzzy-buffer](https://github.com/tzachar/cmp-fuzzy-buffer)
